src/utils.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2025 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""OpenDKIM charm utils."""
+
+import os
+import re
+from pathlib import Path
+
+
+def update_logrotate_conf(
```

Comment:
I think we can simplify this method, considering everything is hard coded now. WDYT?

Reply:
Ok